Jenkinsfile

```diff
@@ -230,6 +230,7 @@ pipeline {
                     post {
                         success { notifySuccessGithub() }
                         failure { notifyFailureGithub() }
+                        always  { cleanWs() }
```

Comment:
Actually not needed. This test is run in a reuseNode agent, that means it shares the workspace with other tests (that also have reuseNode true in its agent section).

When the pipeline has ended, this shared workspace is cleaned once for all these reuseNode tests.

Reply:
Hmm, now I got it, thanks, I wasn't sure if the process were like that, I'll remove ASAP.